src/cpu/cpu_batch_normalization_list.cpp

```diff
@@ -53,6 +57,7 @@ const std::map<pk_impl_key_t, std::vector<impl_list_item_t>> &impl_list_map() {
             CPU_INSTANCE_X64(jit_uni_tbb_batch_normalization_fwd_t<avx512_core>)
             CPU_INSTANCE_X64(jit_uni_tbb_batch_normalization_fwd_t<avx2>)
             CPU_INSTANCE_X64(jit_uni_tbb_batch_normalization_fwd_t<sse41>)
+            CPU_INSTANCE_AARCH64_ACL(acl_batch_normalization_fwd_t)
```

Comment:
@diaena @kawakami-k Is it fine to dispatch into this implementation when both ACL and asimd/sve_512 is available? In convolution list ACL is placed under sve_512 implementations:
https://github.com/oneapi-src/oneDNN/blob/7981216b8341b8603e54472f5a0dd7a12ef9cf67/src/cpu/cpu_convolution_list.cpp#L101-L105

Reply:
As far as I could tell, they cover different memory layouts and ISAs. However, it's a very good point thank you, so I've switched them around in case I missed something.

Reply:
@jondea 
Thank you for correcting the order of the listings. I think it's good to consistent with other primitive lists.

Reply:
BTW, 
I will pull the fixed code for the CI failure https://cloud.drone.io/oneapi-src/oneDNN/1407/7/2
I'm sorry for inconvenience.